zaza/charm_tests/lifecycle/tests.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Code for managing various lifecycle functionality via Zaza."""
+
+import logging
+import os
+
+import zaza.model as model
+
+
+class UpgradeCharmsToPath(object):
+    """Perform charm upgrades to a local file path."""
+
+    test_runner = 'direct_with_args'
+
+    def run(self, charms):
```

Comment:
This function should return a truthy value, as otherwise the check [here](https://github.com/openstack-charmers/zaza/pull/512/files#diff-b301d37c9b8243658d226beff7a6d1635a96ac15343d79300928e82c8b744989R129) will fail.

Reply:
Good catch! Updated in the latest PR